docs/daily-usage.md

```diff
@@ -165,6 +171,8 @@ It's available by default on the [development][] environment, where you can brow
 
 **⚠️ DO NOT USE IT IN PRODUCTION ENVIRONMENTS ⚠️** (I had to say it).
 
+**Note:** For more information, check [Debugging Python code](#debugging-python-code)
```

Comment:
wdb is a different debugger, completely unrelated to VSCode, so this is more exact:

```suggestion
**Note:** Remember that you can instead [use the integrated debugger in Visual Studio Code](#debugging-python-code)
```